lms/djangoapps/organizations/views.py

```diff
@@ -152,6 +153,30 @@ def users(self, request, pk):
 
                     response_data.append(user_data)
             return Response(response_data, status=status.HTTP_200_OK)
+        elif request.method == 'DELETE':
+            user_ids = request.DATA.get('users')
+            if not user_ids:
+                return Response({"detail": _('users parameter is missing.')}, status.HTTP_400_BAD_REQUEST)
+            try:
+                user_ids = map(int, user_ids.split(','))
+            except ValueError:
+                return Response({
+                    "detail": _('users parameter must be comma separated list of integers.')
+                }, status.HTTP_400_BAD_REQUEST)
+
+            users_removed = 0
+            organization = self.get_object()
+            for user_id in user_ids:
+                try:
+                    user = User.objects.get(id=user_id)
+                except ObjectDoesNotExist:
+                    continue
```

Comment:
I am not sure about the requirements, but the count of users not removed might also make sense.